plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/persistence/LibraryClasspathContainerSerializer.java

```diff
@@ -124,6 +136,9 @@ public LibraryClasspathContainer loadContainer(IJavaProject javaProject, IPath c
       Gson gson = new GsonBuilder().create();
       SerializableLibraryClasspathContainer fromJson =
           gson.fromJson(reader, SerializableLibraryClasspathContainer.class);
+      if (fromJson == null) {
+        return null;
+      }
```

Comment:
If the file contents are deleted/modified externally or the class into which we try to serialize changes in an incompatible way.